examples/stm32h7/src/bin/eth.rs

```diff
@@ -108,16 +108,11 @@ fn main() -> ! {
     info!("Hello World!");
 
     info!("Setup RCC...");
-    let mut rcc_config = RccConfig::default();
-    rcc_config.sys_ck = Some(Hertz(400_000_000));
-    rcc_config.pll1.q_ck = Some(Hertz(100_000_000));
-    let config = Config::default().rcc(rcc_config);
-
-    let mut p = embassy_stm32::init(config);
+    let mut p = embassy_stm32::init(config());
 
     // Constrain and Freeze clock
 
-    let mut rcc = Rcc::new(&mut p.RCC, RccConfig::default());
+    let mut rcc = Rcc::new(&mut p.RCC, rcc::Config::default());
```

Comment:
This is initializing RCC twice (though it already did that before...?), it shouldn't do so :S